tools/crm_mon.c

```diff
@@ -655,7 +655,10 @@ reconnect_after_timeout(gpointer data)
     }
 #endif
 
-    out->info(out, "Reconnecting...");
+    if (output_format == mon_output_console) {
+        out->info(out, "Reconnecting...");
+    }
+
```

Comment:
We should avoid checking the format directly whenever possible. We should use messages for all these. However it feels like there's a logical affinity here, so it might be better to create a new output method to use with all of them and make it a no-op for everything but console (the log formatter could maybe log it at debug level).

Maybe something like ->transient()

Reply:
Do we need to make a separate output type for console in order to do that? Right now there isn't one. It seems like it would just be a duplicate of `output_text.c` with names changed, other than implementing `out->transient()`.

Reply:
On second thought, by the time we were to implement an `output_console` type, that would defeat the purpose of having `transient()`. `out->info()` would effectively be transient for the console.

Reply:
There is a console format -- it is implemented solely by crm_mon, in crm_mon_curses.c. Which reminds me, that's one of the reasons it's always good to have a default implementation for custom messages, because the console can use that if there's no explicit implementation in crm_mon_curses.c.